wagtailio/static/sass/components/_theme-toggle.scss

```diff
@@ -0,0 +1,96 @@
+.theme-toggle {
+    $root: &;
+    margin-inline-end: $gutter;
+    display: flex;
+    align-items: center;
+    margin-top: 30px;
+
+    @include media-query(medium) {
+        margin-top: 0;
+    }
+
+    @include media-query(large) {
+        margin-inline-end: ($gutter * 1.5);
+    }
+
+    &__label {
+        font-size: 12px;
```

Comment:
Could use the `.tag-label` class which would give you 12px font size but in rems - https://github.com/wagtail/wagtail.org/blob/main/wagtailio/static/sass/base/_typography.scss#L145